server.js

```diff
@@ -94,7 +95,7 @@ module.exports = function() {
         },
         validate: {
           payload: {
-            currency: Joi.string().min(3).max(3).required(),
+            currency: Joi.any().valid(currencyFor.stripe),
```

Comment:
Did you try it without required?

I remember some weirdness with min(3).max(3) would pass as undefined if I didn't say required.